dahee/services/likeService.js

```diff
@@ -0,0 +1,14 @@
+const likeDao = require('../models/likeDao');
+
+const newLike = async (userId, postId) => {
+  try {
+    const createLike = await likeDao.createLike(userId, postId);
+    return createLike;
```

Comment:
```suggestion
    return likeDao.createLike(userId, postId);
```
굳이  두 줄로 작성될 필요없는 코드네요!